lib/base.js

```diff
@@ -1147,7 +1147,7 @@ class Request extends EventEmitter {
       throw new RequestError(`SQL injection warning for param '${name}'`, 'EINJECT')
     }
 
-    if (arguments.length === 1) {
+    if (arguments.length <= 1) {
```

Comment:
Might as well make it clear.

```suggestion
    if (arguments.length < 2) {
```

Reply:
Fair enough